web/src/components/tasks/business-formation/business/BusinessStep.test.tsx

```diff
@@ -1045,5 +1079,156 @@ describe("Formation - BusinessStep", () => {
       await attemptApiSubmission(page);
       expect(screen.getByRole("alert")).toHaveTextContent(Config.formation.fields.addressState.label);
     });
+
+    it("isVeteranNonprofit", async () => {
+      const page = await getPageHelper({ legalStructureId: "nonprofit" }, { isVeteranNonprofit: undefined });
+      await attemptApiSubmission(page);
+      expect(screen.getByRole("alert")).toHaveTextContent(Config.formation.fields.isVeteranNonprofit.label);
+      expect(screen.getByText(Config.formation.fields.isVeteranNonprofit.error)).toBeInTheDocument();
+    });
+
+    it("hasNonprofitBoardMembers", async () => {
```

Comment:
[pebble] I know what these tests are doing, but typically we're trying to be a little more explicit with our test names. Is there a better name for the tests in this file than just the associated formation data field names?

Reply:
Applies to all new tests in this file.

Reply:
I think this is [sand] aka follow-up refactor because every test in this "required field" describe section is just named on the field name alone. These new ones are following the pattern and it should be an independent separate change if we want to make this across the whole file (and across not just this file but the tests for the other steps as well)

Reply:
Will refactor this separately.